webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/editablemapping/EditableMappingService.scala

```diff
@@ -571,7 +571,7 @@ class EditableMappingService @Inject()(
         dataLayer = segmentationLayer,
         cuboid = request.cuboid(segmentationLayer),
         segmentId = request.segmentId,
-        scale = request.scale,
+        voxelSizeFactor = request.scale,
```

Comment:
Is this field really called `request.scale` and sent this way by the frontend? Shouln't we change this to `voxelSizeFactor` instead (backend & frontend)? This would make the naming more consistent in my opinion